CoreGraphNode/src/au/gov/asd/tac/constellation/graph/node/plugins/DefaultPluginInteraction.java

```diff
@@ -54,33 +55,49 @@
 
     private static final String STRING_STRING_FORMAT = "%s: %s";
 
-    public DefaultPluginInteraction(final PluginManager pluginManager, PluginReport pluginReport) {
+    public DefaultPluginInteraction(final PluginManager pluginManager, final PluginReport pluginReport) {
         this.pluginManager = pluginManager;
         this.pluginReport = pluginReport;
     }
 
+    public PluginManager getPluginManager() {
+        return pluginManager;
+    }
```

Comment:
Actually are you able to remove this function? I think I added it intending to use it in the unit tests but didn't end up doing so

Reply:
@antares1470 I've re-pointed my branch to yours. I had to point it to `master` to trigger the unit tests to run. You should get the credit for this work not me.